src/lib/utils.js

```diff
@@ -1,5 +1,86 @@
+const path = require('path');
+const fs = require('fs');
+const moment = require('moment');
+const isEmpty = require('lodash/isEmpty');
+
 const UTILS = {
+  getUserDirectory() {
+    const homePath = process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE;
+    return path.join(`${homePath}/geek-lab_local`);
+  },
+
```

Comment:
made a read and a write generic so that within 2 functions i can handle read / writes for config and metrics file, and eventually any other '.json' file inside of `~/geek-lab_local' we need